.changeset/tender-gifts-battle.md

```diff
@@ -0,0 +1,20 @@
+---
+"@transloadit/sort-object-by-prio": minor
```

Comment:
These should be majors

Reply:
I agree, but since these are all on `0.x`, the minor versions now behave as a major.